locales/templates/app.pot

```diff
@@ -27,6 +27,10 @@ msgctxt "firefox_tv_brand_name"
 msgid "Firefox TV"
 msgstr ""
 
+msgctxt "firefox_tv_brand_name_short"
+msgid "Firefox"
```

Comment:
In the other string you add here, %1$s  is used for the brand name.
Shouldn't it be the case here as well?

Reply:
I think in this case, "Firefox" is the short name, and "Firefox TV" is the full name - I'd consider them separate strings, depending on which we want to use.

Reply:
oh ok, the other string's comment reads "Content description for the Exit Firefox button.   %1$s will be replaced
with the Firefox brand name." so I assumed they were the same.
I can go ahead and merge then, thanks